### DISCUSSION TITLE
Multi-line code snippet

### DISCUSSION THREAD

**Discussion by nitindhar7** [saadeghi/daisyui#1647] [Q&A] [answered]
How does one use the `Code mockup` component for a multi-line code snippet? My use-case is right on my website here: https://snoozable.io/ in the hero section. The `$ curl --request POST ...` is a string with newline characters and I would like to render that.

**Comment by saadeghi** [comment, marked as answer]
Anything inside `<pre>` will respect the newline character.  
Here's an example:
https://play.tailwindcss.com/gv9ub6rEFj
